src/fetcher.js

```diff
@@ -37,6 +37,7 @@ const parseRDFaDOM = require('./rdfaparser').parseRDFaDOM
 const RDFParser = require('./rdfxmlparser')
 const Uri = require('./uri')
 const Util = require('./util')
+const serialize = require('./serialize')
```

Comment:
I assume you're adding `const serialize = require('./serialize')` to `fetcher.js` because you want to fix the usage of `$rdf.serialize` on line 590?
Do you mind adding that fix to this PR?
(Also, while you're in `fetcher.js`, can you also fix the global usage of `$rdf.Util` on line 867?)